docs/support_playbook.md

```diff
@@ -98,61 +98,57 @@ Make sure you know which part you are amending. Add `\r\n\r\n` for carriage retu
 ApplicationForm.find(_id).update!(becoming_a_teacher: 'new text', subject_knowledge: 'new_text', audit_comment: 'Updating grade following a support request, ticket ZENDESK-LINK')
 ```
 
-## Offers
-
-### Make or change offer
+## Courses and course locations 
 
-If the current application status is `awaiting_provider_decision` use MakeAnOffer service.
-
-If the current application status is `offer` use ChangeOffer service.
+### Changing a course or course location 
 
-### Change provider/course
+A provider may request that a candidate be placed on an different course, or a different site.
 
-Sometimes a provider has no room for an accepted candidate and they refer them to another provider. In this case we create a new ApplicationChoice for the new course_option by following these steps:
-
-- Get the old application choices to withdraw later
-
-For example by candidate’s email:
+- Find the course
 
 ```ruby
-old_application_choice = ApplicationForm.find_by(candidate_id: Candidate.find_by(email_address: 'example@email.com').id).application_choices.first
+course = Course
+  .includes(:provider)
+  .where(
+    code: _course_code, 
+    provider: Provider.find_by(code: _provider_code), 
+    recruitment_cycle_year: RecruitmentCycle.current_year
+  ).first 
 ```
 
-- Add course after submission
-
-Find the course option for the new course. As the course code is not unique make sure to include the provider details.
+- Find the course option
 
 ```ruby
-course_option = CourseOption
+new_course_option = CourseOption
+  .includes(:site)
   .where(
-    course_id: Course.where(code: _course_code, provider: Provider.find_by(code: _provider_code)),
-    recruitment_cycle_year: RecruitmentCycle.current_year
+    course_id: course,
+    study_mode: _study_mode,
+    site: { id: _site_id }
   )
-).first
 ```
 
-- Find the application form
-
-Eg -
+- Find the current application choice
 
 ```ruby
-application_form = old_application_choice.application_form
+application_choice = ApplicationChoice.find(_application_choice_id)
 ```
 
-- Create a new ApplicationChoice associated with the application form
-
-Then submit the new application choice to the provider by calling a service:
+- Finally, update the course option
 
 ```ruby
-SupportInterface::AddCourseChoiceAfterSubmission.new(application_form: application_form, course_option: course_option).call
+application_choice.update_course_option_and_associated_fields!(
+  new_course_option, 
+  audit_comment: "Zendesk ticket #_ticket_number - update course")
```

Comment:
I was trying to put this into a service and I think this method was put in place in order to easily update the offered_course and not the applied one, so we need to also specify `course_option` as part of other fields in order to get the course to be updated sucesfuly.


```suggestion
application_choice.update_course_option_and_associated_fields!(
  new_course_option, 
  other_fields: { course_option: new_course_option },
  audit_comment: "Zendesk ticket #_ticket_number - update course")
```

Reply:
🤔 When I did some of these tickets I did not pass in `other_fields: { course_option: new_course_option }` - everything seemed to be ok. If 'other_fields' is a required param in this scenario then seems like this should not be 'optional'?  Perhaps this is a sign we need a different method with a different (more explicit) signature?